packages/core/src/core/plan/filter.ts

```diff
@@ -131,18 +131,13 @@ export const filterInvalidChanges = (
       getChangeData(diffGraph.getData(nodeId)).elemID
     )
 
-    const createDependencyErr = (causeID: ElemID, droppedID: ElemID): DependencyError => {
-      const message = `Dropped changes to ${
-        droppedID.getFullName()
-      } due to an error in ${causeID.getFullName()}`
-      return {
-        causeID,
-        elemID: droppedID,
-        message,
-        detailedMessage: message,
-        severity: 'Error' as SeverityLevel,
-      }
-    }
+    const createDependencyErr = (causeID: ElemID, droppedID: ElemID): DependencyError => ({
+      causeID,
+      elemID: droppedID,
+      message: 'Element cannot be deployed due to an error in its dependency',
+      detailedMessage: `${droppedID.getFullName()} cannot be deployed due to an error in its dependency ${causeID.getFullName()}. Please resolve that error and try again.`,
+      severity: 'Error' as SeverityLevel,
```

Comment:
maybe we can modify to the below so it would fail in compile time?
```suggestion
      severity: 'Error' as const,
```

Reply:
It is not my code, I have no problem to change this one but it will make us inconstant 

<img width="668" alt="image" src="https://user-images.githubusercontent.com/63018798/204358078-33e07a3d-287d-4ba9-982c-6c8cd2d8de72.png">

Do you want me to change any appearance of this?

Reply:
Ignore it for now, and SHIP IT